recipe/meta.yaml

```
@@ -11,7 +11,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```

As nothing has changed with the other packages, leaving the `build/number` as-is

```suggestion
  number: 0
```